ui-kit/src/main/kotlin/org/p2p/uikit/components/edittext/UiKitEditText.kt

```diff
@@ -0,0 +1,48 @@
+package org.p2p.uikit.components.edittext
+
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.res.use
+import android.content.Context
+import android.os.Parcelable
+import android.util.AttributeSet
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetUiKitEdittextBinding
+import org.p2p.uikit.utils.focusAndShowKeyboard
+import org.p2p.uikit.utils.inflateViewBinding
+
+class UiKitEditText @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+) : ConstraintLayout(context, attrs) {
+    private val binding: WidgetUiKitEdittextBinding by lazy(LazyThreadSafetyMode.NONE, ::inflateViewBinding)
+
+    val values: UiKitEditTextValues = UiKitEditTextValues(binding)
```

Comment:
now UiKitEditText consists of 2 parts:
1) `values` - get immutable values from this view
2) `mutator` - set new listeners / texts / tags and other stuff

all this was for less garbage in a whole class, now the code that modifies the view is in a separate class
you also have a visible contract what can you do with this view

Reply:
before that, this class was like this everything mixed with anything else
<img width="780" alt="Screenshot 2023-07-23 at 17 22 09" src="https://github.com/p2p-org/key-app-android/assets/100791035/d19a116d-35bc-4a11-80d8-936b704b6c06">
<img width="780" alt="Screenshot 2023-07-23 at 17 21 59" src="https://github.com/p2p-org/key-app-android/assets/100791035/e2e8b991-1ce0-4110-a360-63287844c506">